oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/registration/SupplierClientRegistrationRepository.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2023 the original author or authors.
```

Comment:
Spring Security prefers to specify a range. Please use `2002-2023`.

Reply:
I have made the necessary change.